test/Groups/subgroups_and_cosets.jl

```diff
@@ -327,6 +327,32 @@ end
    
 end
 
+@testset "Complement classes" begin
+   # solvable group
+   G = symmetric_group(4)
+   N = pcore(G, 2)[1]
+   @test length(complement_class_reps(G, N)) == 1
```

Comment:
This is also one of the functions we should rename to something like **`representative_of_XXXX** (possibly `complement_class_reps` can stay as an alias for convenience of GAP users who might guess this name?)